lib/errors.js

```diff
@@ -72,6 +78,27 @@ Errors.prototype = {
     },
 
     /**
+     * Formats message according to message data and _verbose flag
+     *
+     * @param {String|Error} message
+     */
+    _prepareMessage: function(message) {
+        if (!this._verbose) {
+            return message;
+        }
+
+        if (typeof message !== 'object') {
+            return this._currentRule + ': ' + message;
+        }
+
+        return 'Error running rule ' + this._currentRule + ': ' +
+            'This is an issue with JSCS and not your codebase.\n' +
+            'Please file an issue about it with stack trace in tracker: ' +
```

Comment:
Maybe it could say: `Please file an issue (with the stack trace below) at\n https://github.com/jscs-dev/node-jscs/issues/new `

Or `about it with the stack trace below in our tracker: `